DBZ-6482 Document TimescaleDB SMT (#4928)
debezium/debezium: documentation/modules/ROOT/pages/transformations/timescaledb.adoc

    "schema.include.list": "_timescaledb_internal", // <10>
    "transforms": "timescaledb", // <11>
    "transforms.timescaledb.type": "io.debezium.connector.postgresql.transforms.timescaledb.TimescaleDb", <12>
    "transforms.timescaledb.database.hostname": "timescaledb", // <13>
    "transforms.timescaledb.database.port": "5432", // <14>
    "transforms.timescaledb.database.user": "postgres", // <15>
    "transforms.timescaledb.database.password": "postgres", // <16>
    "transforms.timescaledb.database.dbname": "postgres" // <17>
  }
}
----
<1> The name of the connector when registered with a Kafka Connect service.
<2> The name of this PostgreSQL connector class.
<3> The address of the TimescaleDB server.
<4> The port number of the TimescaleDB server.
<5> The name of the TimescaleDB user.
<6> The password for the TimescaleDB.
<7> The name of the TimescaleDB database to connect to.
<8> The topic prefix for the TimescaleDB server/cluster, which forms a namespace and is used in all the names of the Kafka topics to which the connector writes, the Kafka Connect schema names, and the namespaces of the corresponding Avro schema when the Avro converter is used.

> <9> Indicates the usage of `pgoutput` logical decoding plug-in.

Reviewer: ```suggestion
<9> Indicates use of the `pgoutput` logical decoding plug-in.
```

> <10> A list of all schemas containing TimescaleDB physicial tables.

Reviewer: ```suggestion
<10> A list of all schemas that contain TimescaleDB physical tables.
```

== Configuration options

The following table lists the configuration options that you can set for the TimescaleDB integration SMT.

.TimescaleDB integration SMT (`TimescaleDB`) configuration options
[cols="30%a,25%a,45%a"]
|===
|Property
|Default
|Description

|[[timescaledb-property-database-hostname]]<<timescaledb-property-database-hostname, `+database.hostname+`>>
|No default
|IP address or hostname of the TimescaleDB database server.

|[[timescaledb-property-database-port]]<<timescaledb-property-database-port, `+database.port+`>>
|`5432`
|Integer port number of the TimescaleDB database server.

|[[timescaledb-property-database-user]]<<timescaledb-property-database-user, `+database.user+`>>
|No default
|Name of the TimescaleDB database user for connecting to the TimescaleDB database server.

|[[timescaledb-property-database-password]]<<timescaledb-property-database-password, `+database.password+`>>
|No default
|Password to use when connecting to the TimescaleDB database server.

|[[timescaledb-property-database-dbname]]<<timescaledb-property-database-dbname, `+database.dbname+`>>
|No default

> |The name of the TimescaleDB database from which to stream the changes.

Reviewer: ```suggestion
|The name of the TimescaleDB database from which to stream changes.
```

|[[timescaledb-property-schema-list]]<<timescaledb-property-schema-list, `+schema.list+`>>
|`_timescaledb_internal`

> |Comma-separated list schema names that contain TimescaleDB raw (internal) data tables.

Reviewer: ```suggestion
|Comma-separated list of schema names that contain TimescaleDB raw (internal) data tables.
```
